packages/docusaurus-utils/src/markdownLinks.ts

```diff
@@ -103,7 +103,7 @@ export function replaceMarkdownLinks<T extends ContentPaths>({
     // This is [Document 1](doc1.md)
     // [doc1]: doc1.md
     const mdRegex =
-      /(?:\]\(|\]:\s*)(?!https?:\/\/|@site\/)<?(?<filename>[^'"\]\s>]+(?:\s[^'"\]\s>]+)*\.mdx?)>?/g;
+      /(?:\]\(|\]:\s*)(?!https?:\/\/|@site\/)(?<filename>[^'")\]\s>]+\.mdx?)/g;
```

Comment:
I don't understand why this PR changes this code and delete snapshots, that looks unrelated to me

Reply:
that was a mistake, about to fix this soon.